cumulus_api/main.py

```diff
@@ -769,11 +683,12 @@ def update_elasticsearch_index(self, data):
         :return: Request response
         """
         record_type = "elasticsearch/change-index"
-        return self.__crud_records(record_type=record_type, verb="post", data=data)
+        return self.__crud_records(record_type=record_type, verb="put", data=data)
```

Comment:
Example shows at post in docs: https://nasa.github.io/cumulus-api/v14.1.0/#change-index
Should this be put?